tools/migrate/blame.py

```diff
@@ -1,8 +1,10 @@
 import argparse
 import json
+from os.path import join
```

Comment:
PS: I prefer to have os.path.join explicit rather than importing join into the global namespace. Just `import os` is good enough. At least that's what more pythonic people have told me to do ;-)